apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/OperationBuilder.kt

```diff
@@ -79,6 +80,9 @@ class OperationBuilder(
         .addSuperinterface(superInterfaceType())
         .maybeAddDescription(operation.description)
         .makeDataClass(operation.variables.map { it.toNamedType().toParameterSpec(context) })
+        .applyIf(useJvmAnnotations) {
+          addAnnotation(JvmOverloads::class)
```

Comment:
I think I misguided you 🤦 . This is adding the annotation on the class itself, not the constructor. So I don't think it'd work?

Reply:
Oops - you're right.  More on that though - what is the best way to test this?  Should be feasible to add a unit test, but i'm not sure what the best approach there is given the existing test infra?

Reply:
You can turn the option on in [`CodegenTests`](https://github.com/apollographql/apollo-kotlin/blob/a1536a269ffe5896f1b5290e55bd361e2d398861/apollo-compiler/src/test/kotlin/com/apollographql/apollo3/compiler/CodegenTest.kt#L271). Pick a directory that has variables with default values (`variable_default_value` sounds like a good one :)) and it will make sure it compiles. You'll need to regenerate the test fixtures with `./gradlew :apollo-compiler:test -DupdateTestFixtures=true` and commit the changed files